src/main/java/core/gui/theme/ImageUtilities.java

```diff
@@ -688,6 +691,21 @@ public static SVGIcon getSvgIcon(String key, int width, int height) {
 					icon.setInterpolation(INTERP_BILINEAR);
 					icon.setAntiAlias(true);
 					icon.setSvgURI(svgURI);
+
+					if (fill != null || stroke != null || opacity != null)
+					{
+						SVGDiagram diagram = icon.getSvgUniverse().getDiagram(svgURI);
+						SVGRoot root = diagram.getRoot();
+
+						StyleSheet ss = new StyleSheet();
+						root.setStyleSheet(ss);
+						if (fill != null)
+							ss.addStyleRule(new StyleSheetRule("fill", "path", null), fill);
+						if (stroke != null)
+							ss.addStyleRule(new StyleSheetRule("stroke", "path", null), stroke);
+						if (opacity != null)
+							ss.addStyleRule(new StyleSheetRule("opacity", "path", null), opacity);
```

Comment:
I don't know if this is intentional, but the letters are no longer displayed:
![Screenshot 2020-06-17 at 20 13 09](https://user-images.githubusercontent.com/114285/84939924-ffd72080-b0d6-11ea-8ab2-5ab9dda21c88.png)

(Intentional or not, I like it! but then I am not a big user of the feature)

Reply:
yes it's intentional, for me it's more clear

Reply:
might be a dumb question but do we really want to keep the printer? I mean who is printing his lineup really :-)

Reply:
> might be a dumb question but do we really want to keep the printer? I mean who is printing his lineup really :-)

yeah, good point! ;-)